### DISCUSSION TITLE
Sticky navbar moves site to top when clicking menu

### DISCUSSION THREAD

**Discussion by 1zboro1** [understrap/understrap-child#386] [Q&A]
Hello,
I have a problem with navbar on this [page](https://bvelo.pl/). After enabling sticky position to navbar when I click menu button while being in other position than top of the site it moves site to the top. How can I prevent this situation? I want it to stop moving site to the top.

Thanks in advance.

**Comment by wpbuilds** [comment]
Hi @1zboro1 - Your question is probably better suited to [https://stackoverflow.com/](https://stackoverflow.com/) than here as the problem you are experiencing is not an issue with the understrap framework itself. You appear to NOT be using the standard Bootstrap 'offcanvas' menu for your navigation. I would advise that you make use of the built in Bootstrap utility classes wherever possible.
